setup_env.sh

```diff
@@ -0,0 +1,14 @@
+
+#Repository for Qt 5.3 
+sudo add-apt-repository --yes  ppa:beineri/opt-qt532-trusty
+sudo apt-get -qq update
+
+# SDL build dependencies
+sudo apt-get -q -y install cmake  gcc-4.9 g++-4.9 libssl-dev libbluetooth3 libbluetooth-dev libudev-dev libavahi-client-dev bluez-tools sqlite3 libsqlite3-dev automake1.11
```

Comment:
@pestOO  it is, this repository should build SDL and run it. SDL's setup env is too complicated and does not exist in GENIVI. When we will add it to GENIVI anf simplify it, we can try to use sdls setup_env against  this one